src/index.html

```diff
@@ -13,5 +13,133 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      
+      <fieldset>
+        <legend>Presonal information</legend>
+        <div>
+          <label for="surname">Surname:</label>
+          <input type="text" id="surname" autocomplete="off"
+          required
+           pattern="\w+">
+        </div>
+        <div>
+          <label for="name">Name:</label>
+          <input type="text" id="name" autocomplete="off"
+          required
+          pattern="\w+">
+        </div>
+        <div>
+          <label for="age">How old are you?</label>
+          <input type="number" id="age"
+          required value="12"
+          min="1"
+          max="100">
+        </div>
```

Comment:
1) no needs to use extra wrappers like `<div>`
2) put `<input>` inside `<label>` to avoid using for/id
3) apply code styling [recommendations](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) all over the page
4) repeat with similar cases
```suggestion
          <label>
            How old are you?
            <input 
              type="number" 
              required 
              value="12"
              min="1"
              max="100"
            >
          </label>
```

Reply:
got it